src/xdocs/config_misc.xml

```diff
@@ -1279,6 +1279,86 @@ void foo(String aFooString,
       </subsection>
     </section>
 
+    <section name="OrderedProperties">
+      <subsection name="Description" id="OrderedProperties_Description">
+          <p>Since Checkstyle 5.8</p>
+          <p>
+              Checks if property keys (left from equal sign) are sorted.
+          </p>
+          <p>
+              Rationale: Unsorted property makes merges more easy. While there are no errors in
+              runtime. Important is the java sort order where 'only.text' before 'only.text.sub'.
+          </p>
+
+      </subsection>
+
+      <subsection name="Properties" id="OrderedProperties_Properties">
+          <table>
+              <tr>
+                  <th>name</th>
+                  <th>description</th>
+                  <th>type</th>
+                  <th>default value</th>
+                  <th>since</th>
+              </tr>
+              <tr>
+                  <td>fileExtensions</td>
+                  <td>file type extension of the files to check.</td>
+                  <td><a href="property_types.html#stringSet">String Set</a></td>
+                  <td>.properties</td>
+                  <td>5.7</td>
+              </tr>
+          </table>
+      </subsection>
+
+      <subsection name="Examples" id="OrderedProperties_Examples">
```

Comment:
Please expand this section with examples of good order and violations.

Reply:
I don't see any changes yet, but I would start off easy before going into difficult examples. You are trying to introduce users to your check and how violation works.